src/main/java/cloud/orbit/actors/extensions/metrics/dropwizard/InstrumentedMessageSerializer.java

```diff
@@ -0,0 +1,53 @@
+package cloud.orbit.actors.extensions.metrics.dropwizard;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Map;
+
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.MetricRegistry;
+
+import cloud.orbit.actors.extensions.MessageSerializer;
+import cloud.orbit.actors.runtime.BasicRuntime;
+import cloud.orbit.actors.runtime.Message;
+import cloud.orbit.actors.runtime.MessageDefinitions;
+
+public class InstrumentedMessageSerializer implements MessageSerializer {
+    private MetricRegistry metricRegistry;

```

Comment:
Could make these final as they are being set in the constructor. Sometimes get a bit of a performance gain doing that with members.